sw/device/tests/sim_dv/pwrmgr_sleep_all_wake_ups_impl.c

```diff
@@ -246,10 +246,14 @@ void cleanup(uint32_t test_idx) {
       CHECK_DIF_OK(dif_pinmux_wakeup_cause_clear(&pinmux));
       break;
     case PWRMGR_PARAM_PINMUX_AON_USB_WKUP_REQ_IDX:
+      CHECK_DIF_OK(dif_usbdev_set_wake_enable(&usbdev, kDifToggleDisabled));
+      // Write again to make sure the first one has already completed.
```

Comment:
It's a bit unfortunate that we have to acknowledge a wakeup in two places.

One thing I'll say, though, is that this is likely a false error. The clock path skew would have to be _enormous_ between the two flops for the pinmux write to be sampled on an earlier cycle than the usbdev write. Our CDC instrumentation is flawed.